### PR TITLE
feat: add swap partition to image

### PR DESCRIPTION
**Pull request by shodan8192**
This commit adds empty swap partition to sdcard image.

### CODE REVIEW

**Review by gtxzsxxk — COMMENTED** [review]
Hello Emil,
Thanks for your contribution!

And I tried the image from the github action. It seems that the swap partition is not set:

```bash
[root@milkv-duo]~# free -h
              total        used        free      shared  buff/cache   available
Mem:          55.3M       12.0M       34.2M       68.0K        9.1M       39.5M
Swap:             0           0           0
```

Maybe we should do some changes to the `/etc/fstab`. And as we all know, with the erase of  the sdcard, the card's life will be shortened. Could you please add some options to the `milkv-duo.conf` to configure whether to enable the swap partition?

And if you want to append the commits, simply direct commit to the `shodan8192:swap`. Your changes there will sync with here.

Thanks again for your patience!

**Comment by shodan8192** [comment]
Yes - swap partition is here, but not enabled by default, like in official sdk. To have it activated, depending on the option in `milkv-duo.conf` I have to alter `S99user` script. Is that OK ?

**Comment by gtxzsxxk** [comment]
Sure, please!

**Comment by shodan8192** [comment]
Sorry for late reply. Please do not merge this yet. I think that init.d script like `S10swap` would be better for swap activation. Also I have another idea, which I describe in Discussions soon.

**Comment by shodan8192** [comment]
Still WIP. In order to make it work, another script is required.

**Comment by gtxzsxxk** [comment]
Got it. When you are finally ready, please click the circle to re-request review:

image

**Comment by shodan8192** [comment]
This PR is ready, but need merge of https://github.com/gtxzsxxk/duo-buildroot/pull/14 in order to work.

**Comment by gtxzsxxk** [comment]
Could you please `mv` the commit in #14 to here? I'd like to play with them all together after the CI gives me the sdcard.img.

**Comment by shodan8192** [comment]
Done, now it's all here.

File: board/milkv/duo/overlay/etc/init.d/S05runonce [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        for file in /etc/runonce.d/*
+        do
+                if [ ! -f "$file" ]
+                then
+                        continue
+                fi
+                
+                if "$file" >> /tmp/runonce.log 2>&1
+                then
+                        mv "$file" "/etc/runonce.d/ran/"
```
Comment (gtxzsxxk) [inline board/milkv/duo/overlay/etc/init.d/S05runonce:18]: Hello~ What about adding a `sync` here after we move the file, to make sure the file has been indeed written back to the filesystem?

Comment (shodan8192) [reply board/milkv/duo/overlay/etc/init.d/S05runonce:18]: It make sense, as a bonus `sync` will also commit writes (if any) made by called script. Let me add this.

**Review by gtxzsxxk — APPROVED** [review]
Thanks for contribution! Let me merge this.

**Pull request merged**